docs/reference/supported-environments.md

```diff
@@ -75,18 +75,35 @@ Ensure the Camunda component versions are compatible with the Helm chart version
 
 The [sizing of a Camunda 8 installation](/components/best-practices/architecture/sizing-your-environment.md) depends on various influencing factors. Ensure to [determine these factors](../components/best-practices/architecture/sizing-your-environment.md#understanding-influencing-factors), and conduct [benchmarking](../components/best-practices/architecture/sizing-your-environment.md#running-experiments-and-benchmarks) to validate an appropriate environment size for your test, integration, or production environments.
 
-#### Volume performance
+### Persistent volumes
 
-As a minimum requirement, the persistent volumes for Zeebe should use volumes with an absolute minimum of 1,000 IOPS. **NFS or other types of network storage volumes are not supported.**
+Camunda supports different types of network storage volumes, including block storage and network file systems (NFS).
 
-To ensure an appropriate sizing, [determine your influencing factors](../components/best-practices/architecture/sizing-your-environment.md#understanding-influencing-factors) (for example, throughput), and conduct [benchmarking to validate an appropriate environment sizing](../components/best-practices/architecture/sizing-your-environment.md#running-experiments-and-benchmarks).
+For details on typical volume usage, refer to these examples:
 
-For details on typical volume type usage, refer to the following examples specific to cloud service providers:
-
-- [Amazon EKS](/self-managed/reference-architecture/kubernetes.md#amazon-eks)
+- [Amazon EKS](/self-managed/reference-architecture/kubernetes.md#amazon-eks-1)
 - [Microsoft AKS](/self-managed/reference-architecture/kubernetes.md#microsoft-aks)
 - [Google GKE](/self-managed/reference-architecture/kubernetes.md#google-gke)
 
+#### Network File Systems
+
+Camunda guarantees support for Amazon Elastic File System (EFS).
+
+If you want to use another NFS, it must meet these requirements:
+
+- Be POSIX-compliant.
+- Never reorder file operations.
+- Be a hard mount, not a soft mount, to avoid SIGBUS errors during a network partition.
```

Comment:
Hard/soft mounts only apply to NFS, as in the NFS software. It's a bit confusing, as NFS means network file system, and is used interchangeably as a "kind of file system" and as a name referring to the [NFS] (https://wiki.archlinux.org/title/NFS) software developed at Sun Microsystems.

I don't know if this means anything for _all_ network file systems though